cityofaustin/joplin
# 4146 loc urls

Zenhub Link: https://app.zenhub.com/workspaces/techstack-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/4146

Deployed: https://joplin-pr-4146-loc-urls.herokuapp.com/admin/login/

API: https://joplin-pr-4146-loc-urls.herokuapp.com/api/graphiql#query=%7B%0A%20%20allPages%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20janisUrls%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A

# Description

- Joplin now has Janis urls for Location Pages
- Joplin now has a basic Location page added to dummy data with fixtures
- Joplin now has a basic test, which makes sure the structure of the janis url is correct. 

# To test 

See deployed link > 
Create a location page > 
No additional info needed > 
Save as Draft >
Hit that api link >
- [ ] See the fake.url link in the json with your new page ? 
- [ ] look right ?

# Checklist:
